Externals/discord-rpc/thirdparty/include/rapidjson/rapidjson.h

```diff
@@ -236,7 +236,7 @@
 #    define RAPIDJSON_ENDIAN RAPIDJSON_BIGENDIAN
 #  elif defined(__i386__) || defined(__alpha__) || defined(__ia64) || defined(__ia64__) || defined(_M_IX86) || defined(_M_IA64) || defined(_M_ALPHA) || defined(__amd64) || defined(__amd64__) || defined(_M_AMD64) || defined(__x86_64) || defined(__x86_64__) || defined(_M_X64) || defined(__bfin__)
 #    define RAPIDJSON_ENDIAN RAPIDJSON_LITTLEENDIAN
-#  elif defined(_MSC_VER) && defined(_M_ARM)
+#  elif defined(_MSC_VER) && (defined(_M_ARM) || defined(_M_ARM64))
```

Comment:
This is also fixed 2 years ago in rapidjson mainline: https://github.com/Tencent/rapidjson/commit/f91405801f88533c609d95f2fcc2d88811544d35

Maybe just update the entire thing to a recent release?